lighthouse-core/test/results/sample.json

```diff
@@ -1,11 +1,13 @@
 {
   "lighthouseVersion": "x.x.x",
```

Comment:
This just updates sample.json to be from an actual run. We've changed the shape of the JSON a lot with the DBW integration and none of those audits were being included here. I needed to add an example for the document.write audit and that was missing, so I updated to the latest format.